src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        Random random = new Random();
+        Colors[] colors = Colors.values();
+        int randomIndex = random.nextInt(colors.length);
+
+        return colors[randomIndex];
     }
```

Comment:
Consider making the 'Random' instance a private field of the 'ColorSupplier' class instead of creating a new 'Random' object each time the 'getRandomColor()' method is called. This will be more efficient as you won't need to create a new 'Random' object for each method call.